test/upload_test_helpers.js

```diff
@@ -224,5 +225,23 @@ export default {
 
   getCloudfrontInvalidateOptions() {
     return s3Opts.cloudfrontInvalidateOptions
+  },
+
+  getS3Object(key) {
+    const s3 = new S3({
+      accessKeyId: s3Opts.AWS_ACCESS_KEY,
+      secretAccessKey: s3Opts.AWS_SECRET_ACCESS_KEY
+    })
+
+
+    return new Promise((resolve, reject) => {
+      s3.getObject({Bucket: s3Opts.AWS_BUCKET, Key: key}, function(err, data) {
```

Comment:
@MikaAK you can do `s3.getObject().promise()` and thus avoid having ot create a new promise yourself